tests/data/validation/enum/enum.go

```diff
@@ -356,6 +446,16 @@ func (j *EnumMyStringTypedEnum) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// Verify checks all fields on the struct match the schema.
+func (plain EnumMyStringTypedEnum) Verify() error {
+	for _, expected := range enumValues_EnumMyStringTypedEnum {
+		if reflect.DeepEqual(v, expected) {
```

Comment:
where does this `v` come from? I don't see it declared anywhere, are you sure it's valid?